docs/components/modeler/desktop-modeler/element-templates/about-templates.md

```diff
@@ -5,7 +5,7 @@ description: "Element templates are a way to extend Camunda Modeler with domain-
 ---
 
 :::note
-Element templates can be used with Desktop Modeler version 1.0+; they are currently available in BPMN diagrams only.
+Element templates are currently available in BPMN diagrams only.
```

Comment:
🙇 Thank you! Is this applicable to older versions of the docs also? (This is currently only in the "next" version).

Reply:
Yes. Is equally applicable to 8.1 and 8.0 branches.